Tween/AnimationCollection.cs

```diff
@@ -35,9 +35,9 @@ public AnimationCollection Add(params AbstractAnimation[] animations)
 		/// <summary>
 		/// Starts playback of the AnimationCollection starting at the first animation
 		/// </summary>
-		/// <param name="onComplete">An optional callback invoked when the AnimationCollection is complete</param>
+		/// <param name="onComplete">A callback invoked when the AnimationCollection is complete</param>
 		/// <param name="onAbort">An optional callback invoked if the AnimationCollection is aborted</param>
-		public override void Play(Action onComplete = null, Action onAbort = null)
+		public override void Play(Action onComplete, Action onAbort = null)
```

Comment:
Why does this require an `onComplete` now rather than optional?

Reply:
Because we have overload function in `AbstractAnimation`:
`Play() { Play(null); }`.